stdlib/contrib/bonitoo-io/alerta/alerta.flux

```diff
@@ -20,7 +20,22 @@ import "strings"
 // `origin` - string - monitoring component.
 // `timestamp` - time - time alert was generated.
 // `timeout` - int - seconds before alert is considered stale.
-alert = (url, apiKey, resource, event, environment="", severity, service=[], group="", value="", text="", tags=[], attributes, origin="InfluxDB", type="", timestamp=now()) => {
+alert = (
+        url,
+        apiKey,
+        resource,
+        event,
+        environment="",
+        severity,
+        service=[],
+        group="",
+        value="",
+        text="",
+        tags=[],
+        attributes,
+        origin="InfluxDB",
+        type="",
+        timestamp=now()) => {
```

Comment:
@BarWeiner1 @scbrickley I'm guessing this is the desired output of this PR? That function params are each put on their own line?

Reply:
Yeah this is desired behavior